internal/agent/install.go

```diff
@@ -300,8 +300,12 @@ func RunInstall(debug bool, options map[string]string) error {
 
 	_, reboot := options["reboot"]
 	_, poweroff := options["poweroff"]
-	installConfig.Reboot = reboot
-	installConfig.PowerOff = poweroff
```

Comment:
Setting these was making the install action to reboot, skipping the AfterInstall hooks that would setup `grubenv` and such.